packages/web-twig/src/Resources/components/FileUploader/README.md

```diff
@@ -109,12 +109,14 @@ HTML input:
 The maximum size of the uploaded file that is validated by the JavaScript plugin can be adjusted.
 The default value is 10 MB.
 To increase the limit for example to 20 MB, add the `maxFileSize` attribute.
-The attribute value should be given in bytes:
+The attribute value should be given in **bytes**:
```

Comment:
Maybe this sentence should be also updated to:
`The attribute value is given in **bytes**`

Remove `should`, because we accept only bytes? Or even replace with `must`?

Reply:
Yes, please do it :-)